library/src/webapp/editor/ckextraplugins/common-wl/js/folder-listing.js

```diff
@@ -153,8 +154,4 @@ $.fn.folderListing = function(options) {
   });
 };
 
-// automatically bind to data-folder-listing divs
```

Comment:
Yes that was my concern too as removing these lines will stop the functionality of html display of widget on Resources folder.